recipe/meta.yaml

```diff
@@ -7,11 +7,15 @@ package:
 source:
   fn: rios-{{ version }}.tar.gz
   url: https://bitbucket.org/chchrsc/rios/downloads/rios-{{ version }}.tar.gz
-  md5: c6d370262a2630ef81deb53b88e5d184
+  sha256: 59fc71a3cfdc2c6de47fd38bc999bdb6de48ce882c5a680dac97118bf86b486f
 
 build:
-  number: 4
-  script: python setup.py install
+  number: 0
+  noarch: python
```

Comment:
You'll need to re-render to get the CI configure to use only CircleCI.

Reply:
Actually how does that work with entry points? We have a couple of command line programs now that use entry points...

Reply:
The Windows version will need .exe's created to run the entry points.

Reply:
Int theory latest `conda` deals with that. I haven't test it thoroughly though.